.php_cs

```diff
@@ -1,5 +1,26 @@
 <?php
 
+// Use PHP-CS-Fixer 2+ if it is available
```

Comment:
Because the change proposed, we should keep only this one, and remove the old one :)